text/experimental/remote-sampling.md

```diff
@@ -0,0 +1,146 @@
+# Remote Sampling for SDK
+
+This is a proposal to add remotely configurable sampling to OpenTelemetry SDK.
+
+## Motivation
+
+Remote sampling configuration is a useful feature that allows users configuring sampling in a central place (e.g. collector) for
+fleet of running OpenTelemetry SDKs. Apart from a central configuration place the remote sampling offers fine-grained
+per-operation sampling configuration.
+
+Remote sampling is already widely used and adopted in other open-source tracing systems such as [Jaeger](https://www.jaegertracing.io/docs/1.23/sampling/).
+
+## Explanation
+
+This draft is related to [0121-config-service](https://github.com/open-telemetry/oteps/blob/main/text/experimental/0121-config-service.md)
+
+Goals:
+
+* supply configuration from a remote source e.g. OpenTelemetry collector to SDK
+* update configuration in SDK at runtime
+* define remote sampler required configuration
+
+## Internal details
+
+```protobuf
+// Sampling configuration for a single service.
```

Comment:
is there an expectation of protobuf? this is not a high throughput endpoint and json would be easier for human readability.